tendrl/node_agent/objects/definition/node_agent.yaml

```diff
@@ -443,183 +202,6 @@ namespace.node_agent:
-      help: "OS"
-    Package:
-      atoms:
-        install:
-          enabled: true
-          inputs:
-            mandatory:
-              - Package.name
-              - Package.pkg_type
-            optional:
-              - Package.version
-          name: "Install Package"
-          help: "Checks if a package is installed"
-          post_run:
-            - tendrl.node_agent.atoms.package.validations.check_package_installed
-          run: tendrl.node_agent.atoms.package.install.Install
-          type: Create
-          uuid: 16abcfd0-aca9-4022-aa0f-5de1c5a742c7
-      attrs:
-        name:
-          help: "Location of the rpm/deb/pypi package"
-          type: String
-        pkg_type:
-          help: "Type of package can be rpm/deb/pip/"
-        state:
-          help: "State can installed|uninstalled"
-        version:
-          help: "Version of the rpm/deb/pypi package"
-          type: String
-      enabled: true
-      help: "package"
-      value: ""
-    Process:
-      atoms:
-        start:
-          enabled: true
-          inputs:
-            mandatory:
-              - Service.config_path
-              - Service.config_data
-          name: "Configure Service"
-          help: "Checks if a service is running"
-          post_run:
-            - tendrl.node_agent.atoms.service.validations.check_service_running
-          run: tendrl.node_agent.atoms.service.configure.Configure
-          type: Update
-          uuid: b90a0d97-8c9f-4ab1-8f64-dbb5638159a3
-      attrs:
-        name:
-          help: "Name of the service"
-          type: String
-        state:
-          help: "Service state can be started|stopped|restarted|reloaded"
-          type: String
-      enabled: true
-    TendrlContext:
-      enabled: True
-      attrs:
-        integration_id:
-          help: "Tendrl managed/generated cluster id for the sds being managed by Tendrl"
-          type: String
-        sds_name:
-          help: "Name of the Tendrl managed sds, eg: 'gluster'"
-          type: String
-        sds_version:
-          help: "Version of the Tendrl managed sds, eg: '3.2.1'"
-          type: String
-        node_id:
-          help: "Tendrl ID for the managed node"
-          type: String
-      value: nodes/$Node_context.node_id/TendrlContext
-      help: "Tendrl context"
-    NodeContext:
-      attrs:
-        machine_id:
-          help: "Unique /etc/machine-id"
-          type: String
-        fqdn:
-          help: "FQDN of the Tendrl managed node"
-          type: String
-        node_id:
-          help: "Tendrl ID for the managed node"
-          type: String
-        tags:
-          help: "The tags associated with this node"
-          type: String
-        status:
-          help: "Node status"
-          type: String
-
-      enabled: true
-      list: nodes/$Node_context.node_id/Node_context
-      value: nodes/$Node_context.node_id/Node_context
-      help: Node Context
     ClusterNodeContext:
```

Comment:
Same as above as this is nothing specific to node-agent and commons can have this as well I feel.